### PR TITLE
added attributes method to get queue attributes

### PR DESCRIPTION
**Pull request by kumaramit01**
I needed this function at work to distribute work from the queue -feel free to merge if this attributes function would be of use in the mainline.

Amit

### CODE REVIEW

File: src/main/scala/com/kifi/franz/SQSQueue.scala [hunk 1]
```diff
@@ -81,6 +72,27 @@ trait SQSQueue[T]{
     p.future
   }
 
+   def attributes(attributeNames:List[String]):Future[Map[String,String]]={
```
Comment (stkem) [inline src/main/scala/com/kifi/franz/SQSQueue.scala]: I would prefer a `Seq[String]` as input, to make it slightly more generically usable.

File: src/main/scala/com/kifi/franz/SQSQueue.scala [hunk 2]
```diff
@@ -8,16 +8,7 @@ import scala.collection.JavaConverters._
...
-  SendMessageRequest,
-  GetQueueUrlRequest,
-  ReceiveMessageRequest,
-  DeleteMessageRequest,
-  SendMessageResult,
-  ReceiveMessageResult,
-  CreateQueueRequest,
-  MessageAttributeValue
-}
+import com.amazonaws.services.sqs.model._
```
Comment (stkem) [inline src/main/scala/com/kifi/franz/SQSQueue.scala]: I'm not a fan of `_` imports. Makes the code harder to reason about.

Comment (kumaramit01) [inline src/main/scala/com/kifi/franz/SQSQueue.scala]: Yeah that was probably my IDE being intelligent or the scalastyle plugin.
I will fix.
Amit

**Comment by stkem** [comment]
@kumaramit01 You have a merge conflict. After that it's good to go in.

**Comment by kumaramit01** [comment]
@stkem -merge conflict has been resolved.

**Pull request merged**